docs/reference/api/connectivity/drivers/EMAC.md

```diff
@@ -0,0 +1,45 @@
+## EMAC
+
+The Ethernet MAC device driver class provides an interface for a network stack to control the Etherner driver. 
+
+### Default driver
+
+If your EMAC driver is the default for the target, define the static get default instance function to return an instance of your EMAC.
+
+### Usage
+
+The EMAC driver class is instantiated during the creation of the network interface. When the network interface is brought up, the network stack powers the EMAC driver.
+
+Steps that the network stack uses to power the EMAC driver:
+
+1. The EMAC memory manager class reference is configured to the driver. 
```

Comment:
Query: Who or what configures the EMAC memory manager class reference to the driver?